### ISSUE TITLE
Can't see "approve or deny any pending permission request" on SharePoint Admin Center

### ISSUE THREAD

**Issue by licypal** [SharePoint/sp-dev-docs#2243]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

Display this  pop up window before deployment that indicates solution package requires permissions approval:

![image](https://user-images.githubusercontent.com/6519992/42853470-81481486-89eb-11e8-8fdd-5ed452695d7f.png)

#### Observed Behavior

The pop up window doesn't show up and the deployment shows error message:

![image](https://user-images.githubusercontent.com/6519992/42852943-41f46d6e-89e8-11e8-9820-2489018e50b9.png)

#### Steps to Reproduce

Follow this tutorial: https://docs.microsoft.com/en-us/sharepoint/dev/spfx/use-aad-tutorial

Also, I don't see any of the following on my SharePoint Admin Center:

![image](https://user-images.githubusercontent.com/6519992/42849758-87e1b9bc-89d9-11e8-838c-8ba0716a4daa.png)

What I see on my SharePoint Admin Center is:

![image](https://user-images.githubusercontent.com/6519992/42850546-0064a86a-89dd-11e8-83c2-283a9b3e9eb2.png)

Also, one thing I noticed is that I am able to see pop up window after removing the following from config/package-solution.json file:

"webApiPermissionRequests": [
      {
        "resource": "Microsoft Graph",
        "scope": "User.ReadBasic.All"
      }
    ]

But, by removing this, I see the following error on running gulp serve --nobrowser:

![error - remote workbench](https://user-images.githubusercontent.com/6519992/42853446-543f62d2-89eb-11e8-912b-e74dd2742970.PNG)

**SPFx version: 1.4.1**

Please let me know if you need additional details. Thanks!

**Comment by licypal** [comment]
An update on this. After updating release preferences to **targeted release for everyone** (Microsoft 365 admin center -> Settings -> Organizational Profile -> Release preferences), I am able see approval page. After approval, I ran gulp serve --nobrowser, I get this error: 

![error](https://user-images.githubusercontent.com/6519992/42895136-cc38765c-8a6d-11e8-9c02-a4d268cd37e7.jpg)

On selecting cancel, I get an additional error:

![error 1](https://user-images.githubusercontent.com/6519992/42895162-e5106162-8a6d-11e8-9a09-5647ccffd8b4.JPG)

On selecting OK, the app crashes and refreshes. Please let me know if there are any addtional settings I need to do. Thanks!

**Comment by mcmynn83** [comment]
Could it be related to this?
https://github.com/AzureAD/azure-activedirectory-library-for-js/issues/580

**Comment by licypal** [comment]
@mcmynn83 Thanks for your suggestion. Tried one approach. Because I am not able to remove accounts not required, I tried opening the browser in incognito window. Now, I don't get any error. I see the following:

![aadhttpclient](https://user-images.githubusercontent.com/6519992/43025195-a9a249d8-8c25-11e8-9496-349b37d21e9f.JPG)

![msgraphclient](https://user-images.githubusercontent.com/6519992/43025196-a9c3af56-8c25-11e8-950a-c4121d2e687f.JPG)
